lisa/tools/systemd_analyze.py

```diff
@@ -78,6 +86,136 @@ def get_boot_time(self, force_run: bool = True) -> ProvisionBootTimeMessage:
         boot_time.provision_time = self.node.provision_time
         return boot_time
 
+    def send_boot_time_messages(
+        self, test_result: "TestResult" = None, test_case_name: str = ""
+    ) -> None:
+        """
+        Send boot time messages. Sends both ProvisionBootTimeMessage and
+        UnifiedPerfMessage notifications.
+        
+        Args:
+            test_result: Optional test result for UnifiedPerfMessage. If None,
+                        only ProvisionBootTimeMessage is sent.
+            test_case_name: Name of the test case for UnifiedPerfMessage.
+        """
+        from lisa import notifier
+        
+        boot_time = self.get_boot_time()
+        boot_time.information.update(self.node.get_information())
+        
+        # Send ProvisionBootTimeMessage
+        notifier.notify(boot_time)
+        
+        # Send UnifiedPerfMessage notifications if test_result is available
+        if test_result is not None:
+            self._send_unified_boot_time_metrics(boot_time, test_result, test_case_name)
+
+    def send_boot_time_metrics(
+        self, test_result: "TestResult", test_case_name: str = ""
```

Comment:
It doesn't need arguments: test_result: "TestResult", test_case_name: str = "", they are always None or Empty.